addons/account/models/account_move.py

```diff
@@ -1471,6 +1465,19 @@ def _move_autocomplete_invoice_lines_values(self):
             if line.product_id and not line._cache.get('name'):
                 line.name = line._get_computed_name()
 
+            # Compute the account before the partner_id
+            # In case account_followup is installted
+            # Setting the partner will get the account_id in cache
+            # If the account_id is not in cache, it will trigger the default value
+            # Which is wrong in some case
+            # It's better to set the account_id before the partner_id
+            # Ensure related fields are well copied.
```

Comment:
why the first letter is capital each time? do you try to make some kind of code?
C
I
S
I
W
I
E

mmhh... not sure of what you're trying to say, mate

Reply:
besides, I liked it more when there was no space after the # :-D